docs/api/covidcast_changelog.md

```diff
@@ -40,6 +40,14 @@ Duplicate survey weights had corrupted historical figures for the following sign
   * `hrr`: 20200406-20200415, 20200430-20200506
   * `msa`: 20200408-20200414, 20200430-20200506
   * `state`: 20200408-20200416, 20200430-20200506
+  
+#### 20 November 2020  
+
+Due to a bug in our data processing system, estimates of the percentage of people reporting that they have been tested for COVID-19 calculated before October 8th were incorrect. 
+We incorrectly treated an answer of “no” as a missing response, which affected the smoothed_tested_14d and smoothed_wtested_14d signals from the fb-survey source. 
```

Comment:
```suggestion
We incorrectly treated an answer of “no” as a missing response, which affected the `smoothed_tested_14d` and `smoothed_wtested_14d` signals from the `fb-survey` source. 
```

Reply:
We typically put source and signal names in monospace font, which the backticks do